src/main/java/frc/robot/hardware/phoenix6/signal/Phoenix6SignalBuilder.java

```diff
@@ -22,6 +23,19 @@ private static StatusSignal<?> cloneWithFrequency(StatusSignal<?> signal, double
 		return signalClone;
 	}
 
+	private static StatusSignal<?> cloneWithCorrectFrequency(
```

Comment:
```suggestion
	private static StatusSignal<?> cloneWithFrequency(
```
or if you don't want the names to be the same, something that shows one is updated by bus chain and one by thread